docker/stable/ubuntu14.04/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:trusty
+MAINTAINER Andrew Schwartzmeyer <andschwa@microsoft.com>
+
+ARG POWERSHELL_RELEASE=v6.0.0-alpha.9
+ARG POWERSHELL_PACKAGE=powershell_6.0.0-alpha.9-1ubuntu1.14.04.1_amd64.deb
```

Comment:
These lines necessitate updating release documentation to include updating these variables.